src/gui/editorwidgets/qgshtmlwidgetwrapper.cpp

```diff
@@ -128,4 +156,15 @@ QString HtmlExpression::evaluate( const QString &expression ) const
   exp.prepare( &mExpressionContext );
   return exp.evaluate( &mExpressionContext ).toString();
 }
+
+QString NeedsGeometryEvaluator::evaluate( const QString &expression )
```

Comment:
Why returning a (constant empty) string here? Shouldnìt it be `void` ?

Reply:
Yes. Fixed